src/frequenz/sdk/microgrid/_power_managing/_power_managing_actor.py

```diff
@@ -63,15 +65,7 @@ def __init__(  # pylint: disable=too-many-arguments
                 `None` when the component category is enough to uniquely identify the
                 component.
             algorithm: The power management algorithm to use.
-
-        Raises:
-            NotImplementedError: When an unknown algorithm is given.
         """
-        if algorithm is not Algorithm.MATRYOSHKA:
-            raise NotImplementedError(
-                f"PowerManagingActor: Unknown algorithm: {algorithm}"
-            )
-
```

Comment:
This change is in the commit that says it is changing the default algorithm, I would move it into the commit adding the new algorithm instead.